### DISCUSSION TITLE
Problem making new accounts/email validation

### DISCUSSION THREAD

**Discussion by anony-0** [ChatTheatre/Community#11] [General]
Hate to pile on with problems, but it seems that there may be some new account problems here in Lazarus-land.

I asked a former player to log in, mostly just to test some things out and hopefully spot any maps I forgot to re-link up, and she didn't get past the log-in stage.

She has an old acct, but the old password didn't work. So she tried a reset, but no email was sent. So she then tried making a completely new acct, and no email validation sent.

And finally I tried making a second acct, and no email validation sent to me either.

So... something seems broken somewhere?

**Comment by ChristopherA** [comment]
I know that @shannona had to set up some mail anti-spam records to get gmail to receive d aims from ChatTheatre.com. 

Shannon, any advice?

**Comment by shannona** [comment]
For ChatTheatre I set up two records, a DMARC and a SPF.

They dramatically improve the delivery of mail.

The SPF is a domain TXT record in DNS that just lists which servers are
allowed to send mail. It's quite easy to set up.

The DMARC record is keypair. You put the public key in a TXT domain record
with a special prefix, and then you adjust your outgoing mail server (exim)
to sign the mail with the private key and also say what prefix to look up.
It's a little harder to set up.

There are docs online for doing both of them (which is what I'd need to
look up to do them again). The SPF record docs are trivial, whereas the
DMARC docs that I could find weren't quite right (as I recall), so I had to
mod them appropriately for the setup.

**Comment by storystafficara** [comment]
We are experiencing the same issue here in Multi Rev, new accounts are not receiving email verification either.
